restconnector/RequestHandler.cpp

```diff
@@ -83,7 +95,6 @@ void RequestHandler::processGET_( const ::zeq::Event& event )
 
     while( blocked_ )
     {
-        while( subscriber_->receive( 0 ) ){}
```

Comment:
AFAIU, this while(blocked){} is busy waiting the reply for the event that is published, right ? RequestHandler::listen_() function listens the message ( which is another thread and executes the image received event ) and sets this variable and busy waiting is done. IMHO, this should be a monitor or conditional variable. There is a wait on a variable.